server/controllers/userRoleController.js

```diff
@@ -0,0 +1,17 @@
+import models from '../models';
+import { userRoleNotification } from '../helpers/email';
+
+const { User } = models;
+
+const updateRole = async (req, res) => {
+  const { role } = req.body;
+  const { username } = req.params;
+  await User.update({ role }, { where: { username } });
```

Comment:
Yes it can fail, and line 10 would still run even if the user doesn't exist or email is not available.
I am going to correct this and push the changes